notebook/lab/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "devDependencies": {
     "css-loader": "^0.23.1",
-    "es6-promise": "^3.0.2",
+    "es6-promise": "^3.1.2",
```

Comment:
Well, that is the thing (in jupyter-js-widgets), we have this `embed-webpack.js` which is not required by `index.js` and only intended to be webpacked.